resources/public/api/conf/2.0/application.yaml

```diff
@@ -0,0 +1,10973 @@
+                notOpenStatus:
+                  $ref: '#/components/examples/notOpenStatusUnauthorized'
+        '403':
+          description: Forbidden
+          content:
+            application/json:
+              schema:
+                allOf:
+                - $ref: '#/components/schemas/lifetimeIsaErrorResponse'
+              example:
+                code: INVESTOR_NOT_FOUND
+                message: The investor details given do not match with HMRC’s records.
+        '409':
+          description: Conflict
+          content:
+            application/json:
+              schema:
+                allOf:
+                - $ref: '#/components/schemas/lifetimeIsaErrorResponse'
+              examples:
+                recordExists:
+                  value:
+                    code: INVESTOR_ALREADY_EXISTS
+                    message: The investor already has a record with HMRC.
+      deprecated: false
+      security:
+        - user-restricted:
+            - write:lisa
+  /lifetime-isa/manager/{lisaManagerReferenceNumber}/accounts:
+    post:
+      tags:
+        - LISA Accounts
+      summary: Create or transfer a LISA account
+      description: |
+        Create a new account you’ve set up for an investor, or transfer an existing account from another LISA provider. If you’re creating a new account you’ll need to create a LISA investor first.
+        
+        ### Test data
+
+        <table style="font-family:Roboto, sans-serif; font-size:14px;">
+            <col width="25%">
+            <col width="40%">
+            <col width="35%">
+            <thead>
+                <tr>
+                    <th>Scenario</th>
+                    <th>Request Payload</th>
+                    <th>Response</th>
+                </tr>
+            </thead>
+            <tbody>
+                <tr>
+                    <td><p>Create request with a valid payload and LISA Manager reference number</p><p class ="code--block">lisaManagerReferenceNumber: <a href="#section/Testing">Use your test user profile</a></p></td>
+                    <td>
+        <pre class="code--block">
+        {
+          "investorId": "9876543210",
+          "creationReason": "New",
+          "accountId": "1234567890",
+          "firstSubscriptionDate": "2017-04-06"
+        }
+        </pre>
+                    </td>
+                    <td><p>HTTP status: <code class="code--slim">201 (Created)</code></p>
+        <pre class="code--block">
+        {
+          "status": 201,
+          "success": true,
+          "data": {
+            "message": "Account created",
+            "accountId": "1234567890"
+          }
+        }
+        </pre>
+                    </td>
+                </tr>
+                <tr>
+                    <td><p>Transfer request with a valid payload and LISA Manager reference number</p><p class ="code--block">lisaManagerReferenceNumber: <a href="#section/Testing">Use your test user profile</a></p></td>
```

Comment:
Once the duplicated Testing section in this file is removed, please update link from`#section/Testing` to `/api-documentation/docs/api/service/lisa-api/2.0#testing`. Thanks